telco-hub/configuration/reference-crs/optional/cert-manager/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
The hub will also need entries under the example-overlay-configs to show how to update the api/ingress endpoints, the issuer url, etc.
Thanks!